test/trento_web/controllers/v1/sap_system_controller_test.exs

```diff
@@ -15,7 +15,8 @@ defmodule TrentoWeb.V1.SapSystemControllerTest do
 
   describe "list" do
     test "should list all sap_systems", %{conn: conn} do
-      %{id: sap_system_id, database_id: database_id} = build(:sap_system)
+      %{id: sap_system_id, database_id: database_id} = insert(:sap_system)
```

Comment:
Could we add a real database and provide the real database_id when inserting the sap system?

Reply:
@EMaksy for more info, initially @CDimonaco created a refence between this `database_id` and the `id` on the database read model.
This was to assure that there couldn't be any sap system without an existing database.
Due some reasons, we removed that constraint initially, for example, to make the tests a bit more flexible.
But if we implement them properly, we could at some point get back that reference, and the tests should continue passing